app/config/AppConfig.scala

```diff
@@ -54,7 +54,7 @@ class AppConfig @Inject()(config: Configuration, servicesConfig: ServicesConfig)
 
   def getPenaltyDetailsUrl: String = {
     if(!isEnabled(CallAPI1812ETMP)) stubBase + "/penalty/details/"
```

Comment:
Could you add the following on to this as well: 
```suggestion
    if(!isEnabled(CallAPI1812ETMP)) stubBase + "/penalty/details/VATC/VRN/"
```